File: Java/Code Related to Arrays/55. Find smallest element in an Array.java
import java.util.Scanner;

class Main {
    public static void main(String [] args) {
        Scanner userInput = new Scanner(System.in);

        System.out.print("Enter the number of elements of the Array: ");
        int size = userInput.nextInt();
        int[] numbers = new int[size];

        for (int i = 0; i < size; i++) {
            System.out.print("Element " + (i + 1) + ": ");
            numbers[i] = userInput.nextInt();
        }

        int smallest = numbers[0];

        for (int i = 1; i < size; i++) {
            if (numbers[i] < smallest) {
                smallest = numbers[i];
            }
        }
        // Method 2

        // for (int num : numbers) {
        //     smallest = Math.min(smallest,num);
        // }
        System.out.println("The Smallest element in the array is: " + smallest);
        userInput.close();
    }
}

File: Python/03. Code Related to Arrays/55. Find smallest element in an Array.py
lengthOfArray = int(input("Enter the length of an Array:"))
array = []

for i in range(lengthOfArray):
    element = int(input(f"Enter {i+1} element :"))
    array.append(element)

if array:
    smallest = array[0]

    for element in array:
        if element < smallest:
            smallest = element
    print("The Smallest element is",smallest)

else:
    print("No elements are entered")

# Method 2

# lengthOfArray = int(input("Enter the length of an Array:"))
# array = []

# for i in range(lengthOfArray):
#     element = int(input(f"Enter {i+1} element :"))
#     array.append(element)
# array.sort()
# print (array[0])

# Method 3
# print(min(array))